backend/molgenis-emx2-graphql/src/main/java/org/molgenis/emx2/graphql/GraphlAdminFieldFactory.java

```diff
@@ -80,19 +86,26 @@ private static Object getUsers(SelectedField selectedField, Database db) {
     int limit = args.containsKey(LIMIT) ? (int) args.get(LIMIT) : 100;
     int offset = args.containsKey(OFFSET) ? (int) args.get(OFFSET) : 0;
     String email = args.containsKey(EMAIL) ? (String) args.get(EMAIL) : null;
+    List<Member> members = MetadataUtils.loadUserRoles((SqlDatabase) db);
```

Comment:
rather not use MetadataUtils here; instead move to SqlDatabase. I try to keep that as a backend (should probably change visibility of MetadataUtils so you would not be tempted to do this)

Reply:
done